.devcontainer/Dockerfile

```diff
@@ -68,8 +68,75 @@ ENV DEBIAN_FRONTEND=noninteractive
 # Java 8 and 11 (not in official repo for focal)
 RUN add-apt-repository -y ppa:openjdk-r/ppa && \
     apt-get update && \
-    apt-get install -y openjdk-8-jdk openjdk-11-jdk
-RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+    apt-get install -y openjdk-11-jdk
+RUN update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
+
+RUN apt-get -y  install apt-transport-https software-properties-common
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0x51716619e084dab9
+RUN add-apt-repository -y 'deb https://cran.rstudio.com/bin/linux/ubuntu focal-cran40/'
+RUN apt update
+RUN apt-get -y install r-base
+
+# copy common dependency installation scripts
+RUN mkdir -p /opt/rstudio-tools/dependencies/linux
+RUN mkdir -p /opt/rstudio-tools/dependencies/tools
+RUN mkdir -p /opt/rstudio-tools/dependencies/common
+COPY ./dependencies/tools/rstudio-tools.sh /opt/rstudio-tools/dependencies/tools/
+
+# copy panmirror project setup (so install-common can install needed deps)
+RUN mkdir -p /opt/rstudio-tools/panmirror
+COPY ./src/gwt/panmirror/src/editor/yarn.lock /opt/rstudio-tools/panmirror/
+COPY ./src/gwt/panmirror/src/editor/package.json /opt/rstudio-tools/panmirror/
+
+# install common dependencies
+# here, we deviate from the typical "./install-common" script and instead
```

Comment:
This makes sense from a docker workflow perspective, but when we adjust dependency scripts (e.g. add new ones, remove old ones), this may break and nothing in the local dev workflow or official builds will catch that. That's probably fine, should be infrequent, and people using the devcontainer will notice and presumably go fix it.

Reply:
That's pretty much what I thought. I'll resolve this, but we can always come back to it later.